can-compute_test.js

```diff
@@ -583,7 +583,8 @@ test("Listening to input change", function(){
 	});
 
 	input.value = 'foo';
-	domDispatch.call(input, "input");
```

Comment:
Did this have to change? I would expect that you could use `can-dom-events.dispatch` for this.

Reply:
Yeah, if it is not changed the test will fail with ```TypeError: Cannot read property 'type' of undefined
    at Object.createEvent (http://localhost:8080/node_modules/can-dom-events/helpers/util.js:20:25)```

Reply:
The test passes with ```input.dispatch('input')``` and fails with ```domDispatch.call(input, "input")```
afer assigning the fn like this ``` var domDispatch = domEvents.dispatch```

Reply:
Ah ok, so would this work?

```js
var domEvents = require("can-dom-events");

domEvents.dispatch(input, "input");
```

Reply:
I try it but it dosen't work!

Reply:
I change it to ```domEvents.dispatch(input, "input");```